README.md

```diff
@@ -68,6 +69,10 @@
 
 Dockerfile to build a [GitLab](https://about.gitlab.com/) container image.
 
+GitLab CE is set up in the Docker image using the [install from source](https://gitlab.com/gitlab-org/gitlab-ce/blob/master/doc/install/installation.md) method as documented in the the official GitLab documentation.
+
+For other methods to install GitLab please refer to the [Official GitLab Installation Guide](https://about.gitlab.com/installation/) which includes a [GitLab image](https://gitlab.com/gitlab-org/gitlab-ce/tree/master/docker).
```

Comment:
I think it should [Docker GitLab image](https://gitlab.com/gitlab-org/gitlab-ce/tree/master/docker).